sorts/bubble_sort.py

```diff
@@ -34,10 +40,13 @@ def bubble_sort(collection):
 
 
 if __name__ == "__main__":
+    import doctest
     import time
 
+    doctest.testmod()
+
     user_input = input("Enter numbers separated by a comma:").strip()
     unsorted = [int(item) for item in user_input.split(",")]
-    start = time.process_time()
+    start = time.time()
```

Comment:
Why make this change?

Reply:
I like `time()` function. Do you think which is better.

Reply:
I think that the original author used [`time.process_time()`](https://docs.python.org/3/library/time.html#time.process_time) for a reason.  And "I like a different function" is not enough of a reason to change the author's intention.

Reply:
alright. I undo it. Please review again.